packages/snaps-controllers/src/snaps/permissions.ts

```diff
@@ -6,15 +6,12 @@ import {
   caveatMappers,
   restrictedMethodPermissionBuilders,
   selectHooks,
+  endowmentCaveatMappers,
+  endowmentPermissionBuilders,
 } from '@metamask/snaps-rpc-methods';
 import type { SnapPermissions } from '@metamask/snaps-utils';
 import { hasProperty } from '@metamask/utils';
 
-import {
-  endowmentCaveatMappers,
-  endowmentPermissionBuilders,
-} from './endowments';
-
 /**
  * Map initial permissions as defined in a Snap manifest to something that can
  * be processed by the PermissionsController. Each caveat mapping function
```

Comment:
Do we not need to move `processSnapPermissions` too?

Reply:
Yea, you're right. Moved all of the related stuff from that file, since its related elements are now in the `snaps-rpc-methods` as well.